webdriver/tests/bidi/script/add_preload_script/invalid.py

```diff
@@ -186,6 +187,49 @@ async def test_params_arguments_channel_include_shadow_tree_invalid_value(bidi_s
         )
 
 
+@pytest.mark.parametrize("contexts", [False, 42, '_UNKNOWN_', {}])
+async def test_params_contexts_invalid_type(bidi_session, contexts):
+    with pytest.raises(error.InvalidArgumentException):
+        await bidi_session.script.add_preload_script(
+            function_declaration="() => {}",
+            contexts=contexts
+        ),
+
+async def test_params_contexts_context_empty_list(bidi_session):
+    with pytest.raises(error.InvalidArgumentException):
+        await bidi_session.script.add_preload_script(
+            function_declaration="() => {}",
+            contexts=[]
```

Comment:
Why should this be invalid? Nothing in the spec says so. Doesn't it mean that the call is a no-op?

Reply:
I think I miss interpreted your previous comment about:
```
Please also add a test that actually checks for list entries.
```

Aaa it seems this was missed in the spec. I am happy to remove on this PR and add it to the tentative I mentioned. But currently adding such `preload script` will do nothing as you mentioned. Will file an issue about this as well.

Reply:
Has this been handled meanwhile?

Reply:
Yes spec change landed - https://github.com/w3c/webdriver-bidi/pull/577